### DISCUSSION TITLE
Update: Ignore this. Can't get authed. Getting 401'd with 'Please refresh the page'. Tried many things.

### DISCUSSION THREAD

**Discussion by rollsveallambchopfig** [UltimaHoarder/UltimaScraper#1085] [Q&A] [answered]
Update: I see now this is a known issue.

I did do a search on here, and I haven't found anything that seems to be the same issue, and any potential solutions I found have not worked.
I've been using this script for a while without issue. I even pulled recently so I am familiar with the new auth.json and I had been getting authed just fine until today. 
I've gone through every cookie I can find, and I used the cookie helper as well. Nothing works. I even used a python debugger to step through the code in case something was getting corrupted from the settings or auth JSON files. It all looks good. 
I turned off 2FA just in case that would make is simpler. 
I saw auth_uid_{auth_id} was set to None in the code. I have that cookie, but not the auth_unique, so I changed the code to send that. Still didn't work.
Is anyone having this same problem and has anyone worked around it?

**Comment by faketucker** [comment]
same here

**Comment by fightcancer2** [comment]
Same issue.

**Comment by purpleemoji** [comment]
Issue just started for me today. All the uath,json info is correct: sess and user_agent were both updated. Cleared cache, rebooted and tried again yet can't get auth'd. OF must have changed something

**Reply by purpleemoji** [reply]
Latest commit fixed my auth issues.

**Reply by fightcancer2** [reply]
Where can I find the latest commit please?

Answer: apparently "commit" means version in this context, so just download the newest version of this code. :)

**Comment by rollsveallambchopfig** [comment, marked as answer]
Was documented and fixed in issue #1078

**Comment by faketucker** [comment]
Yesterday still worked, but today it happened again

**Reply by fightcancer2** [reply]
Same
